packages/backend/routes/v1/chat/auth.ts

```diff
@@ -88,16 +88,19 @@ authRouter.get('/oauth-callback', async (req, res) => {
             try {
                 await xprisma.connections.upsert({
                     where: {
-                        id: String(req.query.t_id),
+                        id: `${account!.accounts!.id}_${String(req.query.t_id)}`,
```

Comment:
@ashutosh-revert 

Could you turn all occurrences of ``${account!.accounts!.id}_${String(req.query.t_id)}`` into a function instead? 

This way we'd only have to make a change once if we decide to update the logic again